libs/platform/core/src/services/storage/default-storage.service.ts

```diff
@@ -1,15 +1,29 @@
-import { inject } from '@spryker-oryx/di';
+import { inject, INJECTOR } from '@spryker-oryx/di';
 import { isObservable, map, Observable, of } from 'rxjs';
 import { IndexedDBStorageService } from './indexed-db-storage.service';
-import { StorageType, StoredValue } from './model';
-import { StorageService } from './storage.service';
+import { StorageType } from './model';
+import { StorageService, StorageStrategy } from './storage.service';
+
+function toObservable<T>(value: Observable<T> | T): Observable<T> {
```

Comment:
Maybe better to move it to utilities?

Reply:
👍

Reply:
actually we have this utility already 🤦 ;)